mindjunkies/lecture/views.py

```
@@ -18,7 +18,10 @@
 from mindjunkies.courses.models import Course, Module
 
 from .forms import LectureForm, LecturePDFForm, LectureVideoForm, ModuleForm
-from .models import Lecture, LecturePDF, LectureVideo
+from .models import Lecture, LecturePDF, LectureVideo, LectureCompletion
+from django.utils.timezone import localtime, now
```

Ruff (0.8.2)

Redefinition of unused `localtime` from line 12

Remove definition: `localtime`

(F811)